all_in_one_seo_pack.php

```diff
@@ -499,3 +499,39 @@ function disable_all_in_one_free() {
 		}
 	}
 }
+
+add_action( 'admin_post_aioseop_upgrade', 'upgrade');

```

Comment:
Let's not add to this file. Ideally we'd start moving things out of it instead and organize better. 
At most, let's not add more than an add_action if needed.

Reply:
I agree. Have used this to include another file that will contain similarly-themed functions.